src/Main.java

```diff
@@ -0,0 +1,45 @@
+// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
+// then press Enter. You can now see whitespace characters in your code.
+public class Main {
+    public static void main(String[] args) {
+        // Press Opt+Enter with your caret at the highlighted text to see how
+        // IntelliJ IDEA suggests fixing it.
+        System.out.printf("Hello and welcome!");
+
+
+        int positiveOrNegativeOrZero = 0;
+        if (positiveOrNegativeOrZero > 0) {
```

Comment:
Deze if statement mag je in een aparte "positiveOrNegativeOrZero" methode zetten. De if/else is op zich goed uitgewerkt, maar het staat nog niet in een methode. De opdracht beschreef een "positiveOrNegative" en een "positiveOrNegativeOrZero" methode. Je hebt nu alleen de tweede